tests/Rules/Deprecations/data/echo-deprecated-magic-method-tostring.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace CheckDeprecatedStaticMethodCall;
+
+$bar = new MagicBar();
+echo $bar;
```

Comment:
it seems this line actually echos the value `a string` from line 15.
is there anything I could/should do differently?